src/picologging/handler.cxx

```diff
@@ -128,8 +131,12 @@ PyObject* Handler_close(Handler *self){
 }
 
 PyObject* Handler_handleError(Handler *self, PyObject *record){
-    // TODO: Develop this behaviour further.
-    PyErr_Print();
+    PyObject* mod = PICOLOGGING_MODULE(); // borrowed reference
+    PyObject* modDict = PyModule_GetDict(mod); // borrowed reference
+    PyObject* raiseExceptions = PyDict_GetItemString(modDict, "raiseExceptions"); // PyDict_GetItemString returns a borrowed reference
+    if (PyBool_Check(raiseExceptions) && PyErr_Occurred()) {
+        PyErr_Print();
```

Comment:
Need to format the exception probably the same way the Formatter does, with all frames.